bin/minifollowups/pycbc_foreground_minifollowup

```diff
@@ -50,7 +50,12 @@ parser.add_argument('--inspiral-data-read-name',
 parser.add_argument('--inspiral-data-analyzed-name',
                     help="Name of inspiral segmentlist containing data "
                          "analyzed by each analysis job.")
-parser.add_argument('--output-map')
```

Comment:
Eek, I accidentally removed this line! This breaks the code. I'll open a PR shortly fixing this!